packages/effect/src/BigInt.ts

```diff
@@ -547,14 +547,89 @@ export const multiplyAll = (collection: Iterable<bigint>): bigint => {
 }
 
 /**
- * Convers a bigint into a number
+ * Takes a `bigint` and returns an `Option` of `number`.
+ *
+ * If the `bigint` is outside the safe integer range for JavaScript (`Number.MAX_SAFE_INTEGER`
+ * and `Number.MIN_SAFE_INTEGER`), it returns `Option.none()`. Otherwise, it converts the `bigint`
+ * to a number and returns `Option.some(number)`.
+ *
+ * @param b - The `bigint` to be converted to a `number`.
+ *
+ * @example
+ * import { fromBigInt } from "effect/Number"
```

Comment:
Example needs updating :)

Last change and I think we can merge this.

Reply:
thank you for your feedback. I fixed it.
by the way,  I wonder how you find some error in comment section 
 Is it just eye-tracking or have some tools for finding some problem ?)

Reply:
The CI scripts check all the examples.